sympy/polys/tests/test_factortools.py

```diff
@@ -27,7 +27,7 @@ def test_dmp_trial_division():
 
 def test_dup_zz_mignotte_bound():
     R, x = ring("x", ZZ)
-    assert R.dup_zz_mignotte_bound(2*x**2 + 3*x + 4) == 32
+    assert R.dup_zz_mignotte_bound(2*x**2 + 3*x + 4) == 6
```

Comment:
Perhaps there could be a couple of more examples, preferable with comments on actual bounds for comparison.